testsuite/cases/pillow.py

```diff
@@ -5,9 +5,6 @@
 from .base import BaseTestCase, root
 
 
-Image.LANCZOS = Image.ANTIALIAS
```

Comment:
Pillow-perf intended to work with all versions of Pillow, not only the latest, and even with versions before `Image.LANCZOS` introduced.

```python
if hasattr(Image, "ANTIALIAS"):
    Image.LANCZOS = Image.ANTIALIAS
```

Reply:
I would have thought this line isn't needed, regardless of the Pillow version, because neither `LANCZOS` or `ANTIALIAS` is used in the rest of this file?

Reply:
It is [actually used](https://github.com/python-pillow/pillow-perf/blob/576e3d6f5a8ee7c34143dd2fb080a93abc8152dd/testsuite/cases/pillow_scale.py#L26)